Add retries to scheduled backups (#586)
percona/pmm-doc: docs/using/backup.md

                - In the second field, select one or more minutes (*00* to *59*).

        - *Retention*: How many backups to keep. For unlimited, use `0` (zero).

>         - *Retry mode*: In case of error, either let PMM retry the backup again ("Auto") or leave it to the user ("Manual").

Reviewer: ```suggestion
        - *Retry mode*: In case of error, either let PMM retry the backup again ("Auto") or do it again yourself ("Manual").
```

Reviewer: On the principle of "talk to the reader"...

Author: Agree, thanks!
